src/components/Swipers/PopularSwiper.tsx

```diff
@@ -1,15 +1,38 @@
 // Import Swiper React components
 import { Swiper, SwiperSlide } from "swiper/react";
+import React, { useEffect, useState } from "react";
 
 // Import Swiper styles
 import "swiper/css";
 import ResturantCard from "../ResturantCard/ResturantCard";
 
-const PopularSwiper = (props: { resturants: { imgUrl: string; name: string; chef: string }[]}) => {
+const PopularSwiper = (props: { resturants: { imgUrl: string; name: string; chef?: string, rating?: string }[]}) => {
+  const [slidesPerView, setSlidesPerView] = useState<number>(1.5);
+  useEffect(() => {
+    const handleResize = () => {
```

Comment:
i saw exactly this function in certain files, write it in helpers file in utils folder and import it